examples/gno.land/p/demo/grc/grc20/bank.gno

```diff
@@ -4,27 +4,10 @@ import (
 	"std"
 	"strconv"
 
-	"gno.land/p/demo/avl"
 	"gno.land/p/demo/ufmt"
 )
 
-// Banker implements a token banker with admin privileges.
-//
-// The Banker is intended to be used in two main ways:
-//  1. as a temporary object used to make the initial minting, then deleted.
-//  2. preserved in an unexported variable to support conditional administrative
-//     tasks protected by the contract.
-type Banker struct {
-	name        string
-	symbol      string
-	decimals    uint
-	totalSupply uint64
-	balances    avl.Tree // std.Address(owner) -> uint64
-	allowances  avl.Tree // string(owner+":"+spender) -> uint64
-	token       *token   // to share the same pointer
-}
-
-func NewBanker(name, symbol string, decimals uint) *Banker {
+func NewBank(name, symbol string, decimals uint) (*Bank, *AdminBanker) {
```

Comment:
I prefer this approach because you can generate your contract-related GRC20 token as a top-level variable in two ways:

```
import "p/grc20"
var Bank, admin = grc20.NewBank("...")
```

or like this:

```
import "p/grc20"
var _, Admin = grc20.NewBank("...")
```

I expect that most, if not all, contracts will eventually want to:
-  expose the safe object publicly
-  maintain a private instance of the admin for administrative tasks

This method allows for a single line of code for the most common initialization.